src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Commands/SaveSearch/UnsubscribeSavedSearchCommand.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using MediatR;
+
+namespace SFA.DAS.FindAnApprenticeship.Application.Commands.SaveSearch;
+
+public record UnsubscribeSavedSearchCommand(Guid Id) : IRequest;
+
+//public class UnsubscribeSavedSearchCommand : IRequest
+//{
```

Comment:
Please could you remove the un-used code or commented ones

Reply:
Addressed in latest commit.